### DISCUSSION TITLE
Adding provision for "and" in split by collection membership.

### DISCUSSION THREAD

**Discussion by nairabhishek73** [glific/glific#2719] [Ideas]
**Is your feature request related to a problem? Please describe.**
Users can be part of multiple collections. When I want to split by collection so that users who are part of collection A and collection B and collection C go to one branch and all other users go to another branch. Then first I need to do a split of A and other, then users who come through this node will be split for B and other, then C and other. So I need to create 3 nodes for this. As users create more and more collections and build more specific targeting, they will need to create more and more nodes.

**Describe the solution you'd like**
An "and" functionality within this node where you can just pick "and" and then multiselect the collections and then that becomes one bucket.

**Describe alternatives you've considered**
NA

**Additional context**
https://discord.com/channels/717975833226248303/971657960726081576/1072788885161136198

**Comment by nairabhishek73** [comment]
@AkhileshNegi @mdshamoon @pankaj-ag

**Comment by mdshamoon** [comment]
Let's keep it on hold for now and check if this usecase is coming up with more NGOs. Meanwhile let's add the details in this issue for all NGOs that can potentially be benefitted from this.
